backend-shared/config/db/injection_test.go

```diff
@@ -1,281 +1,93 @@
 		}
-		err = dbq.CreateGitopsEngineCluster(ctx, &gitopsEngineCluster)
-		if !assert.NoError(t, err) {
-			return
-		}
 
-		gitopsEngineInstance := GitopsEngineInstance{
-			Gitopsengineinstance_id: "test-fake-engine-instance-id-0",
-			Namespace_name:          "test'fake'namespace",
-			Namespace_uid:           "test-fake-namespace-0",
+		gitopsEngineInstance := db.GitopsEngineInstance{
+			Gitopsengineinstance_id: "test-fake-engine-instance-id",
+			Namespace_name:          "test-fake-namespace",
+			Namespace_uid:           "test-fake-namespace-5",
 			EngineCluster_id:        gitopsEngineCluster.Gitopsenginecluster_id,
 		}
-		err = dbq.CreateGitopsEngineInstance(ctx, &gitopsEngineInstance)
-		if !assert.NoError(t, err) {
-			return
-		}
-		retrieveGitopsEngineInstance := &GitopsEngineInstance{
-			Gitopsengineinstance_id: gitopsEngineInstance.Gitopsengineinstance_id,
-		}
-		err = dbq.GetGitopsEngineInstanceById(ctx, retrieveGitopsEngineInstance)
-		if !assert.NoError(t, err) {
-			return
-		}
-		assert.Equal(t, gitopsEngineInstance.Namespace_name, retrieveGitopsEngineInstance.Namespace_name)
-	}
-	assert.NoError(t, err)
-}
 
-func TestClusterCredentialWrongInput(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	clusterCredentials := ClusterCredentials{
-		Clustercredentials_cred_id: "test-cluster-creds-input",
-		Host:                       "host'sInput'",
-	}
-
-	err = dbq.CreateClusterCredentials(ctx, &clusterCredentials)
-	if !assert.NoError(t, err) {
-		return
-	}
-	retrievedClusterCredentials := &ClusterCredentials{
-		Clustercredentials_cred_id: clusterCredentials.Clustercredentials_cred_id,
-	}
-	err = dbq.GetClusterCredentialsById(ctx, retrievedClusterCredentials)
-	if !assert.NoError(t, err) {
-		return
-	}
-
-	assert.Equal(t, clusterCredentials.Host, retrievedClusterCredentials.Host)
-	assert.NoError(t, err)
-}
-
-func TestManagedEnviromentWrongInput(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	clusterCredentials := ClusterCredentials{
-		Clustercredentials_cred_id:  "test-cluster-creds-test-1",
-		Host:                        "host",
-		Kube_config:                 "kube-config",
-		Kube_config_context:         "kube-config-context",
-		Serviceaccount_bearer_token: "serviceaccount_bearer_token",
-		Serviceaccount_ns:           "Serviceaccount_ns",
-	}
-
-	err = dbq.CreateClusterCredentials(ctx, &clusterCredentials)
-	assert.NoError(t, err)
-	{
-		managedEnvironment := ManagedEnvironment{
-			Managedenvironment_id: "test-managed-env-1",
-			Clustercredentials_id: clusterCredentials.Clustercredentials_cred_id,
-			Name:                  "test'env",
```

Comment:
Think a few of the old injection_tests were not ported over to the new tests. For example, here we are creating a managed environment with an name that might cause issues with the SQL logic (if we were doing it wrong).